src/main.ts

```diff
@@ -69,13 +69,45 @@ app.config.globalProperties.$filters = {
     }
     return featureValue;
   },
+  groupFeatures(featureHierarchy: any) {
+    let features =[] as any;
+    let  featuresValue = '';
+    const featuresList = {} as any;
+    if (featureHierarchy) {
+      featureHierarchy.forEach((feature: any) => {
+        const featureSplit = feature ? feature.split('/') : [];
+        features.push(featureSplit[1]);
+        features = [...new Set(features)];
+        features.forEach((x: any)=>{
+          if(featureHierarchy){
```

Comment:
Why do we need this check? We are already checking for featureHierarchy on line 76.

Reply:
I might have added that by mistake, removed it and it is working fine.